protected/modules/OphCiExamination/migrations/m170920_050016_patient_medication_assignment_view.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+class m170920_050016_patient_medication_assignment_view extends CDbMigration
+{
+    public function createView($view_name, $select)
+    {
+        $this->dbConnection->createCommand('create view ' . $view_name . ' as ' . $select)->execute();
+    }
+
+    public function dropView($view_name)
+    {
+        $this->dbConnection->createCommand('drop view ' . $view_name)->execute();
+    }
+
+    public function up()
+    {
+        $this->createView('medication_examination_events', <<<EOSQL
```

Comment:
heredoc, clever

Reply:
Its probably a better standard to use for SQL statements & queries than just using strings.